files/setup-ssh.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+
```

Comment:
Could you add a note that this file is a copy from skuba? And if somebody wants to update it, it should be done in skuba and then synced

Reply:
tbh is not a copy from skuba directly as its modified by ourselves to write to a file and accept a parameter...I guess I could just pointthat is based off that file?

Reply:
Or modify the file in skuba, too (if that makes sense). But keeping it as it is is fine for me